ArticleTemplates/assets/scss/type/_media-gallery.scss

```diff
@@ -1,5 +1,9 @@
 .garnett--type-media[data-content-type='gallery'],
 .is_advertising.garnett--type-guardianlabs[data-content-type='gallery'] .article {
+    .advert-slot--mpu {
+        margin: 0;
```

Comment:
I think ads on other pages have a top/bottom margin

Reply:
I added the same top and bottom margin as other adverts but i didn't know if it looked a bit strange with the gap at the bottom? 
![Simulator Screen Shot - iPhone Xʀ - 2019-08-20 at 15 00 30](https://user-images.githubusercontent.com/19835654/63353883-5b236f80-c35b-11e9-982d-a784e0036c18.png)

Reply:
Yeah maybe 
```
margin: 0;
margin-bottom: 32px;
```

So we have the same margin above and below the ad?